main/src/ca/uwaterloo/flix/language/ast/Ast.scala

```diff
@@ -751,6 +751,8 @@ object Ast {
     object Decl {
       case object Trait extends Decl
 
+      case object Type extends Decl
```

Comment:
I. think we should sort the members of Decl by alphabet.

Reply:
yes, ofc. Somehow I thought it was. Dumb oversight on my end. It's sorted now